pkg/cli/subcommands/check/command.go

```diff
@@ -236,14 +235,11 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 	cmd.Flags().BoolVarP(&cliParams.formatTable, "table", "", false, "format aggregator and check runner output as an ascii table")
 	cmd.Flags().StringVarP(&cliParams.breakPoint, "breakpoint", "b", "", "set a breakpoint at a particular line number (Python checks only)")
 	cmd.Flags().BoolVarP(&cliParams.profileMemory, "profile-memory", "m", false, "run the memory profiler (Python checks only)")
-	cmd.Flags().BoolVar(&cliParams.fullSketches, "full-sketches", false, "output sketches with bins information")
```

Comment:
We need to fix the unusedness of this option, but even if we didn't need to support it anymore, the check command still needs to accept it for backwards compatibility. Would it be possible to keep this line?

Reply:
good point about backward compatibility ! I'll add it back